rasa/nlu/featurizers/dense_featurizer/_lm_featurizer.py

```diff
@@ -0,0 +1,802 @@
+# flake8: noqa
+# WARNING: This module will be dropped before Rasa Open Source 3.0 is released.
+#          Please do not make changes to this module, and instead makes changes to
+#          `LanguageModelFeaturizerGraphComponent`
+#          This module is a workaround to defer breaking changes due to the architecture
+#          revamp in 3.0.
```

Comment:
This is the old `lm_featurizer.py` but renamed. The only change is the highlighted comment.